### PR TITLE
Reorganize repo to priorize plugin

### PR DESCRIPTION
**Pull request by Giovannini**

### CODE REVIEW

**Comment by Giovannini** [comment]
test this please

File: example/fibonacci/src/test/scala/io/giovannini/EvenFibonnacciTest.scala [hunk 1]
```diff
@@ -7,7 +7,7 @@ class EvenFibonnacciTest extends WordSpec {
   "fibonnacci" should {
```
Comment (khand19) [inline example/fibonacci/src/test/scala/io/giovannini/EvenFibonnacciTest.scala]: 🙅 List(1) did not equal List(0)

File: example/fibonacci/src/main/scala/io/giovannini/Main.scala [hunk 2]
```diff
@@ -4,13 +4,11 @@ object Main extends App {
 
   val result = EvenFibonnacci.sumFilterFibonacciUnder(4000000, _ % 2 < 1)
```
Comment (khand19) [inline example/fibonacci/src/main/scala/io/giovannini/Main.scala:5]: 🙅 [warning] @deprecated now takes two arguments; see the scaladoc.

File: example/fibonacci/src/test/scala/io/giovannini/EvenFibonnacciTest.scala (hunk 1 above)
Comment (khand19) [inline example/fibonacci/src/test/scala/io/giovannini/EvenFibonnacciTest.scala]: 🙅 List(1) did not equal List()

**Pull request merged**